Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings

```diff
@@ -155,11 +155,11 @@
 "subscriptions.list.no_message" = "Nenhuma mensagem";
 "subscriptions.list.sent_an_attachment" = "enviou um ficheiro";
 "subscriptions.list.date.yesterday" = "Ontem";
-"subscriptions.list.actions.read" = "Lido"; // TODO
-"subscriptions.list.actions.unread" = "Não lido"; // TODO
-"subscriptions.list.actions.favorite" = "Favorite"; // TODO
-"subscriptions.list.actions.unfavorite" = "Unfavorite"; // TODO
-"subscriptions.list.actions.hide" = "Fechar"; // TODO
+"subscriptions.list.actions.read" = "Lido";
+"subscriptions.list.actions.unread" = "Não lido";
+"subscriptions.list.actions.favorite" = "Marcar como favorito";
```

Comment:
Do you have a "shorter" way to say that? The UI for this button is very small, preferable just one word.

In Portuguese we are using "Favoritar"

Reply:
Hope the new PR is more within the idea